ckine/figures/figureCommon.py

```diff
@@ -303,3 +309,174 @@ def calc_res(sc):
     # find result of minimization where both params are >= 0
     res = least_squares(calc_res, guess, bounds=(0.0, np.inf))
     return res.x
+
+
+def import_pMuteins():
+    """ Loads CSV file containing pSTAT5 levels from Visterra data for muteins. """
+    path = os.path.dirname(os.path.dirname(__file__))
+    data = pds.read_csv(join(path, "data/Monomeric_mutein_pSTAT_data.csv"), encoding="latin1")
+    data["RFU"] = data["RFU"] - data.groupby(["Cells", "Ligand"])["RFU"].transform("min")
+    for conc in data.Concentration.unique():
+        data = data.replace({"Concentration": conc}, np.round(conc, decimals=9))
+
+    return data
+
+
+def getMutAff():
+    """Returns adjusted binding rates for all muteins in dictionary"""
+
+    mutaffret = {
+        'IL2-060 monomeric': [0.19, 7.32],
+        'Cterm IL-2 monomeric WT': [0.54, 4.29],
+        'Cterm IL-2 monomeric V91K': [0.69, 9.56],
+        'IL2-109 monomeric': [0.71, 7.32],
+        'IL2-110 monomeric': [9.48, 7.32],
+        'Cterm N88D monomeric': [1.01, 24.9]
+    }
+
+    return mutaffret
+
+
+mutaff = getMutAff()
```

Comment:
Why not just assign the dict to this global rather than the separate function? You can assume nobody will mess with it.